src/lib/permissions/proposals/policies/policyStatusDraftNotViewable.ts

```diff
@@ -36,6 +37,14 @@ export async function policyStatusDraftNotViewable({
       }
     });
     return newPermissions;
+  } else if (preComputedSpacePermissionFlags?.deleteAnyProposal) {
+    const allowedSpaceWideProposalPermissions: ProposalOperation[] = ['delete', 'view', 'archive', 'unarchive'];
```

Comment:
Constants should be defined at the top of the file

Reply:
Sounds good, just relocated all of these arrays across proposal policies outside their respective functions